region/brinstar/red/Beta Power Bomb Room.json

```diff
@@ -272,10 +272,24 @@
       "link": [1, 3],
```

Comment:
10: Intermediate can be reduced to 1 hit with morph + can dodge while shooting
11: missile kill can be increased to 2 without dodge while shooting, and reduced to 0 with dodge while shooting and morph

Reply:
Done, although I'm unclear on the discrepancy between 10 and 11... I guess missiles are slightly stronger than spazer/wave?

Reply:
Much stronger